optaplanner-core/src/test/java/org/optaplanner/core/impl/util/ScopeUtils.java

```diff
@@ -0,0 +1,45 @@
+package org.optaplanner.core.impl.util;
+
+import org.optaplanner.core.impl.phase.scope.AbstractPhaseScope;
+import org.optaplanner.core.impl.phase.scope.AbstractStepScope;
+import org.optaplanner.core.impl.solver.scope.SolverScope;
+
+public final class ScopeUtils {
```

Comment:
A) For clarity, can we rename it to ScopeTestUtils?

B) Or just put these methods in PlannerTestUtils instead?

Reply:
I prefer B), so it's easier to find all these test helper methods, but I don't mind A).